python/nano/src/bigdl/nano/pytorch/inference/optimizer.py

```diff
@@ -804,27 +804,31 @@ def obtain_manager(input_model):
                                                             precision="fp32")
             return _context_manager
 
-        def join_manager(manager, new_manager):
+        def join_manager(manager1, manager2):
             def is_bf16(x):
                 return isinstance(x, AutocastContextManager)
-            if (is_bf16(manager) ^ is_bf16(new_manager)) is True:
+            if (is_bf16(manager1) ^ is_bf16(manager2)) is True:
```

Comment:
why throw error? just use `autocast` and report warning?

Reply:
Done. Have used warning to replace error.